lib/doc_viewer.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const getFullPath = path =>
+  require('path').resolve(__dirname, path);
```

Comment:
Зависимости лучше подгружать в отдельной строке от кода, в начале файла все
```suggestion
const { resolve } = require('path');
const getFullPath = path => resolve(__dirname, path);
```
или даже через bind можно теперь